sklearn/manifold/tests/test_spectral_embedding.py

```diff
@@ -19,6 +19,15 @@
 from sklearn.utils._testing import assert_array_almost_equal
 from sklearn.utils._testing import assert_array_equal
 
+try:
+    from pyamg import smoothed_aggregation_solver  # noqa
+
+    pyamg_available = True
+except ImportError:
+    pyamg_available = False
+skip_if_no_pyamg = pytest.mark.skipif(
+    not pyamg_available, reason="PyAMG is required for the tests in this function."
+)
```

Comment:
Nitpick.
```suggestion
try:
    from pyamg import smoothed_aggregation_solver  # noqa
    pyamg_available = True
except ImportError:
    pyamg_available = False

skip_if_no_pyamg = pytest.mark.skipif(
    not pyamg_available, reason="PyAMG is required for the tests in this function."
)
```

Reply:
black complained about not have an empty line after import